### DISCUSSION TITLE
136

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#68] [General]
# 潮流周刊第136期 - 深圳湾桥

封面来源于晚上在深圳湾散步拍的一张大桥照片，对面就是香港，海风很凉爽，相比杭州很不一样的体会。

https://weekly.tw93.fun/posts/136-%E6%B7%B1%E5%9C%B3%E6%B9%BE%E6%A1%A5

**Comment by Jie-Huangi** [comment]
终于找到了这个链接。很久之前看到你推荐了一个关于经济的视频，找了半天，终于发现了。
